src/Cohesity.Powershell/Scripts/InterfaceGroup/Get-CohesityInterfaceGroup.ps1

```diff
@@ -0,0 +1,33 @@
+function Get-CohesityInterfaceGroup {
+    <#
+        .SYNOPSIS
+        Get interface group.
+        .DESCRIPTION
+        The Get-CohesityInterfaceGroup function is used to get interface group.
+        .NOTES
+        Published by Cohesity
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        Get-CohesityInterfaceGroup
+    #>
+    [CmdletBinding()]
+    Param()
```

Comment:
Add a new line after the parameter declaration.

Reply:
Fixed